clippy_utils/src/consts.rs

```diff
@@ -161,12 +167,18 @@ impl<'tcx> Hash for Constant<'tcx> {
             Self::Int(i) => {
                 i.hash(state);
             },
+            Self::F16(f) => {
+                f64::from(f).to_bits().hash(state);
+            },
             Self::F32(f) => {
                 f64::from(f).to_bits().hash(state);
             },
             Self::F64(f) => {
                 f.to_bits().hash(state);
             },
+            Self::F128(f) => {
+                f.to_bits().hash(state);
```

Comment:
What should I do here? Functions aren't yet available to convert and use `f128` for everything instead.